### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dirkkelly**
* https://github.com/BetaNYC/citygram-nyc/issues/20 Badges
* https://github.com/BetaNYC/citygram-nyc/issues/15 New Relic Exception
* https://github.com/BetaNYC/citygram-nyc/issues/12 Travis CI

This is the includes https://github.com/codeforamerica/citygram/pull/159

We now have 

* [Travis CI](https://travis-ci.org/BetaNYC/citygram-nyc/builds/51315698)
* [Code Climate](https://codeclimate.com/github/BetaNYC/citygram-nyc)
* [Gemnasium](https://gemnasium.com/BetaNYC/citygram-nyc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
+__Citygram__ is a geographic notification platform designed to work with open government data. It allows residents to designate area(s) of a city they are interested in and subscribe to one or more topics. When an event for a desired topic occurs in the subscriber's area of interest, a notification (email, SMS, or  [webhook](http://en.wikipedia.org/wiki/Webhook)) is delivered. Citygram is a [Code for America](https://github.com/codeforamerica) project by the [Charlotte](http://team-charlotte.tumblr.com/) and [Lexington](http://teambiglex.tumblr.com/) teams for the [2014 fellowship](http://www.codeforamerica.org/geeks/our-geeks/2014-fellows/).
 
 ### Why are we doing this?
 
 We believe that there is an opportunity to help residents better understand what’s going on in their area, when it’s going to happen, and why. By providing timely information to residents in areas that are relevant to them, the city can be proactive instead of reactive, build trust through transparency, and increase civic engagement across the board.
 
-### How to contribute
```
Comment (excepttheweasel) [inline README.md]: Did you intend to remove this?

Comment (dirkkelly) [inline README.md]: I did not, cheers.

**Comment by excepttheweasel** [comment]
+r

**Comment by dirkkelly** [comment]
https://citygram-nyc-staging.herokuapp.com/

**Pull request merged**